Lib/inspect.py

```diff
@@ -382,7 +382,12 @@ def isfunction(object):
         __defaults__    tuple of any default values for arguments
         __globals__     global namespace in which this function was defined
         __annotations__ dict of parameter annotations
-        __kwdefaults__  dict of keyword only parameters with defaults"""
+        __kwdefaults__  dict of keyword only parameters with defaults
+        __dict__        namespace which is supporting arbitrary function attributes
+        __closure__     None or tuple of cells
+        __qualname__    fully qualified name of this function
+        __module__      name of the module the function was defined in or None
```

Comment:
Could we sort or group these? E.g. `__qualname__` and maybe `__module__` make sense next to `__name__`